docs/api/wrappers/shimmy_wrappers.md

```diff
@@ -40,7 +41,8 @@ while env.agents:
 
 
 To load an OpenSpiel game of [backgammon](https://github.com/deepmind/open_spiel/blob/master/docs/games.md#backgammon):
-```python
+```python notest
+#TODO ImportError: cannot import name 'OpenspielCompatibilityV0' from 'shimmy'
```

Comment:
Oh it was changed to be OpenSpiel instead of Openspiel, minor typo before but same comment as above with installing extras for shimmy